bundles/org.openhab.ui/web/src/pages/settings/file-definition-mixin.js

```diff
@@ -0,0 +1,97 @@
+import Vue from 'vue'
+import Clipboard from 'v-clipboard'
+
+Vue.use(Clipboard)
+
+export default {
+  data () {
+    return {
+      ObjectType: Object.freeze({
```

Comment:
As this is a constant it is better to set it in the created method, data is mainly meant for reactive data:

```js
this.OBJECT_TYPE = ....
```

Reply:
thanks for the tip. done.